index.html

```diff
@@ -183,5 +194,79 @@ <h3>Skills</h3>
         </div>
       </div>
     </section>
+    <!-- contactme -hana & henry-->
+    <section class="contactme">
+      <h2>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <form
+        action="https://formspree.io/f/maykropo"
+        method="post"
+        class="contact-form"
+      >
+        <div class="firstlastname">
+          <input
+            class="form-element"
+            id="firstname"
+            type="text"
+            name="name"
+            placeholder="First Name"
+            maxlength="30"
+            required
+          />
+          <input
+            class="form-element"
+            id="secondname"
+            type="text"
+            name="name"
+            placeholder="Last Name"
+            maxlength="30"
+            required
+          />
+        </div>
```

Comment:
- [ ] As per the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md); there should not be two input fields for `first name` and `last name`. Rather consider using one input field for `full name`. This is an exception from the Figma design. Kindly fix this issue.

- ***Requirement***
![Screenshot from 2022-12-08 19-12-21](https://user-images.githubusercontent.com/60083437/206455942-be68bafc-f21a-445d-9f92-56ad293915d0.png)

- ***Your Design***
![Screenshot from 2022-12-08 19-12-50](https://user-images.githubusercontent.com/60083437/206456116-5f40db2d-6ac5-4131-8af7-df72c47d2d18.png)